src/arrayUtils.js

```diff
@@ -0,0 +1,68 @@
+if (typeof maybeExport === 'undefined') {
```

Comment:
I agree there's a risk of keeping them in sync, yeah, but it seems better to me. As it is, that code is added twice, in the html and js, and it's not needed in either, and looks confusing.

If they aren't in sync, we might be missing an export, but that's an easy to debug problem.

We should add a comment on each, though, that their exports are in another place, that's a good idea.